src/components/entity/__internal__/entity-header.component.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledEntityHeader, StyledEntityHeaderContainer } from './entity.style';
+
+const EntityHeader = ({
+  fontSize = 16, title, adornments, menu
+}) => {
+  return (
+    <StyledEntityHeaderContainer data-element='entity-header'>
+      <StyledEntityHeader fontSize={ fontSize }>{title}</StyledEntityHeader>
+      {adornments}
+      {menu}
+    </StyledEntityHeaderContainer>
+  );
+};
+
+EntityHeader.propTypes = {
+  /** Entity title  */
+  title: PropTypes.string,
+  /** Title font-size */
+  fontSize: PropTypes.oneOf([14, 16]),
```

Comment:
I think it would be better to allow the user provide `14px` or `16px` instead of just a number "14" or "16" because the number won't tell us what is it `px, % ...` etc
Why don't we allow user to provide a custom value? eg. '32px'?